preset/azurite/preset_test.go

```diff
@@ -0,0 +1,144 @@
+package azurite_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/orlangure/gnomock/preset/azurite"
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/orlangure/gnomock"
+)
+
+func TestPreset_Blobstorage(t *testing.T) {
+	t.Parallel()
+
+	for _, version := range []string{"latest"} {
+		t.Run(version, testBlobStorage(version))
+	}
+}
+
+func testBlobStorage(version string) func(*testing.T) {
+	return func(t *testing.T) {
+		p := azurite.Preset(
+			azurite.WithVersion(version),
+		)
+		c, err := gnomock.Start(p)
+
+		defer func() { assert.NoError(t, gnomock.Stop(c)) }()
+
+		assert.NoError(t, err)
+
+		connString := fmt.Sprintf(azurite.ConnectionStringFormat, azurite.AccountName, azurite.AccountKey, c.Address(azurite.BlobServicePort), azurite.AccountName)
+
+		ctx := context.Background()
+
+		azblobClient, err := azblob.NewClientFromConnectionString(connString, nil)
+		assert.NoError(t, err)
+
+		containerName := "foo"
+		_, err = azblobClient.CreateContainer(ctx, containerName, nil)
+		assert.NoError(t, err)
+
+		pager := azblobClient.NewListBlobsFlatPager(containerName, nil)
+		pages := 0
+		for pager.More() {
+			resp, err := pager.NextPage(context.TODO())
```

Comment:
Nitpick: `context.TODO` should probably be replaced with `context.Background` and reused whenever a context is required.

Reply:
Changed in https://github.com/orlangure/gnomock/pull/837/commits/ed2a1eba02172d405bf72098a110c332c91c60c5